ci/build-all-tasks.yml

```diff
@@ -0,0 +1,40 @@
+# This YAML is intended for building ALL tasks and publish them as an artifact 
+# to be used later for Tasks Deployment train
+
+steps:
+# Clean
+- checkout: self
+  clean: true
+
+# Use node 8, npm 5
+- task: NodeTool@0
+  displayName: Use node 8
+  inputs:
+    versionSpec: 8.x
+
+- script: npm i -g npm@6.9.0 --force
+  displayName: Use npm version 6.9.0
+
+# npm install
+- script: npm install
+  displayName: npm install
+
+# Build
+- script: node make.js build
+  displayName: Build tasks
+
+# Stage tasks individually into the package directory
+- script: node ./ci/stage-package.js true individually
+  displayName: Stage tasks individually into the package directory
+
+# Stage all the tasks into a single zip for upload
+- script: node ./ci/stage-package.js true
+  displayName: Stage all the tasks into a single zip for upload
+
+# Publish artifact
+- task: PublishBuildArtifacts@1
+  displayName: Publish package artifact
+  inputs:
+    pathToPublish: _package/AllTasks.zip
```

Comment:
Isn't should be _package/tasks.zip since the archive's name is [hardcoded](https://github.com/microsoft/azure-pipelines-tasks/blob/master/ci/ci-util.js#L15)?